src/playlist-selectors.js

```diff
@@ -375,5 +375,5 @@ export const lowestBitrateCompatibleVariantSelector = function() {
   const playlistsWithVideo =
     playlists.filter(playlist => parseCodecs(playlist.attributes.CODECS).videoCodec);
 
-  return playlistsWithVideo[0] || playlists[0];
+  return playlistsWithVideo[0] || null;
```

Comment:
instead of returning null, you could call the last bandwidth selector with the current context
` return playlistsWithVideo[0] || lasBandwidthSelector.call(this);` I think this would also remove the need to for the temporary `selectedMedia`